.github/rpr.sh

```diff
@@ -24,6 +24,7 @@ branch=$(curl -s https://api.github.com/repos/containous/traefik/pulls/$pr | jq
 
 clean ()
 {
+    git checkout $initial
```

Comment:
If you call `rmpr` while you are checked out on your remote branch, `rmpr` will try to delete the branch you are on. We have to checkout another branch before.